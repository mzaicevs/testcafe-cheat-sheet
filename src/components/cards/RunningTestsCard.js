import React from "react";
import { runningTest } from "../../snippets";
import CodeCard from "../../base_components/CodeCard";

const RunningTestsCard = () => (
  <CodeCard
    cardTitle={runningTest.title}
    codeSnippet={{
      code: runningTest.code,
      language: runningTest.language
    }}
  />
);

export default RunningTestsCard;
