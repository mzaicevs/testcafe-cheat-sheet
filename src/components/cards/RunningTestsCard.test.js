import React from "react";
import { shallow } from "enzyme";
import RunningTestsCard from "./RunningTestsCard";

describe("InstallCard", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<RunningTestsCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
