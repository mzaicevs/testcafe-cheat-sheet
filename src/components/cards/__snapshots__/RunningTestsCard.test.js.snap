// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstallCard matches snapshot 1`] = `
<CodeCard
  cardTitle="Running a test on multiple browsers"
  codeSnippet={
    Object {
      "code": Array [
        "testcafe chrome,firefox,edge src/tests/my.e2e.test.js",
      ],
      "language": "plaintext",
    }
  }
/>
`;
